src/test/compile-fail/crate-path-non-absolute.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
```

Comment:
Can we please collect the tests for this feature in some kind of directory named after the rfc? I would suggest `{compile-fail,run-pass,ui}/rfc2126-crate-paths`.